articles/policies/rate-limits-mgmt-api.md

```diff
@@ -0,0 +1,97 @@
+
+| Tenant Type | Rate Limit (per second) | Rate Limit (per minute) |
+| - | - | - |
+| Free or Trial | 10 | 120 |
+| Enterprise (Non-production) | 10 | 120 |
+| Enterprise (Production) | 50 | 1000 |
+| Developer or Developer Pro (created before 05-21-2020) | 50 | 1000 |
+
+Each endpoint is configured with a bucket that defines:
+
+-  Request limit
+-  Rate limit window (per second, per minute, per day, etc.)
+
+```text
+bucket:
+    size: x
+    per_minute: y
+```
+
+For example, the above states that, for the given bucket, there is a maximum request limit of `x` per minute, and for each minute that elapses, permissions for `y` requests are added back. In other words, for each `60 / y` seconds, one additional request is added to the bucket. This occurs automatically until the bucket contains the maximum permitted number of requests.
+
+For some API endpoints, the rate limits are defined per bucket, so the origins of the call do not influence the rate limit changes. For other buckets, the rate limits are defined using different keys, so the originating IP address is considered when counting the number of received API calls.
+
+::: note
+If you are using an API endpoint **not** listed below and you receive rate limit headers as part of your response, see [Anomaly Detection](/anomaly-detection) for more information.
+:::
+
+The aforementioned rate limits include calls made via [Rules](/rules) and are set **by tenant** and not by endpoint.
+
+The following Auth0 Management API endpoints return rate limit-related headers. For additional information about these endpoints, please consult the [Management API explorer](/api/management/v2).
+
+| Endpoint | Path | Rate Limit (per second) | Rate Limit (per minute) |
+| - | - | - | - |
+| Read or write users | `GET /api/v2/users` | 40 | 800 |
+| | `POST /api/v2/users` | | |
+| Read logs | `GET /api/v2/logs` | 10 | 200 |
+| | `GET /api/v2/user/{id}/logs` | |
+| Read clients | `GET /api/v2/clients` | 5 | 100 |
+| | `POST /api/v2/clients/{id}` | |
+| Read connections | `GET /api/v2/connections` | 10 | 100 |
+| | `POST /api/v2/connections/{id}` | |
+| Delete device credentials | `DELETE /api/v2/device-credentials/{id}` | 5 | 100 |
+| All other endpoints | | 10 | 150 |
+
+## Self-service subscription limits
+
+| Endpoint | Path | Rate Limit (per second) | Rate Limit (per minute) | Rate Limit (per day) |
+| - | - | - | - | - |
+| Verify custom domain | `POST /api/v2/custom-domains{id}/verify` | n/a | 5 | n/a |
+| Send test email | `POST /api/v2/email-send-test` | 1 | n/a | n/a |
+| Register dynamic client | `POST /oidc/register` | 5 | n/a | n/a |
+| Rotate signing keys | `POST /api/v2/keys/signing/rotate` | n/a | n/a | 5 |
+| Create tenant | `POST /api/v2/tenants` | 5 | n/a | n/a |
+| Read connection status | `GET /api/v2/connections/{id}/status` | 15 | n/a | n/a |
+| Create tenant invitation | `POST /api/v2/tenants/invitations` | n/a | 5 | n/a |
```

Comment:
This applies for everyone, not only self-service.

Reply:
Merged tables